src/sentry/types/group.py

```diff
@@ -61,6 +61,7 @@ class GroupSubStatus:
 
 
 class PriorityLevel(IntEnum):
+    OK = 0
```

Comment:
i don't think we should change the issue priority enum here, instead use the `DetectorPriority` which is a superset of this that includes `OK`